python-pytz has been removed from Slackware -current because
not needed with Python 3.11
This patch prevents pip check from indicating the lack of
pytz as dependency required by OWSLib in -current
--- ./requirements.txt	2024-04-05 23:42:52.864498274 +0200
+++ ./requirements.txt	2024-04-05 23:43:13.322499915 +0200
@@ -1,6 +1,5 @@
 dataclasses; python_version < '3.7'
 lxml
 python-dateutil>=1.5
-pytz
 pyyaml
 requests>=1.0
